konf/site.yaml

```
@@ -6,11 +6,16 @@ env:
   - name: SENTRY_DSN
     value: https://37706e022f2841448dbb094990420522@sentry.is.canonical.com//23
 
-  - name: SEARCH_API_KEY
+  - name: FLASK_GOOGLE_SEARCH_API_KEY
     secretKeyRef:
       key: google-custom-search-key
       name: google-api
 
+  - name: FLASK_SECRET_KEY
```

```suggestion
  - name: SECRET_KEY
```
Why don't we call it `SECRET_KEY` here? Then we don't need to alias it in app.py

---

[Here](https://docs.google.com/document/d/1zOmyP6FyMQMhGNOs69O7bVGpPSi110z-KQ8cG_Phe1s/edit#heading=h.ehuw67mkj6pz) is the answer to your question.